<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>公安单兵式网络舆情监控系统</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tabWidget QTabBar{
	border-bottom:  1px solid rgb(177, 177, 177);
}
#tabWidget QTabBar::tab{
	width: 7em;
	height: 1.5em;
	margin-left: 1px;
	background-color: rgb(244, 244, 244);
	border: 1px solid rgb(177, 177, 177);
	border-right: 1px solid rgb(213, 213, 213);
	border-top: 1px solid rgb(213, 213, 213);
	border-top-left-radius: 10px;
}
#tabWidget QTabBar::tab:selected{
	height: 1.6em;
	border-bottom: 0;
}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>首页(&amp;A)</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>83</y>
       <width>711</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="HotNewsWidget" name="hotnews" native="true">
        <property name="styleSheet">
         <string notr="true">#hotnews a:hover{
    color: rgb(0, 85, 255);
    font: bold;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>危情查看(&amp;B)</string>
    </attribute>
    <widget class="DateTree" name="dateTree">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>84</y>
       <width>200</width>
       <height>370</height>
      </rect>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QListWidget" name="weiQingList">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>84</y>
       <width>484</width>
       <height>370</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>单兵作战(&amp;C)</string>
    </attribute>
    <widget class="danbinTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>171</width>
       <height>381</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>离线作战(&amp;D)</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>舆情分析(&amp;E)</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>我的收藏(&amp;F)</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>系统管理(&amp;G)</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>0</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#quit{
	background-color: #66CCFF;
	border: 0;
}
#quit:hover{
	background-color: rgba(255,75,75,0.8);
	font-weight: 500;
	color: rgb(244,255,255);
}</string>
   </property>
   <property name="text">
    <string>退出系统(&amp;H)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>700</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.392, y2:0.427, stop:0 rgba(255, 255, 255, 0), stop:1 rgba(255, 255, 125, 141));
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>55</y>
      <width>190</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>你好，管理员，今天是：</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>700</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="MainWindow.qrc">:/img/images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="getDate" name="date">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>55</y>
      <width>122</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1970年01月01日</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>getDate</class>
   <extends>QLabel</extends>
   <header>getdate.h</header>
  </customwidget>
  <customwidget>
   <class>HotNewsWidget</class>
   <extends>QWidget</extends>
   <header>hotnewswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateTree</class>
   <extends>QTreeWidget</extends>
   <header>datetree.h</header>
  </customwidget>
  <customwidget>
   <class>danbinTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>danbintreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>quit</tabstop>
 </tabstops>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
